config/kinoite-packages.yml

```diff
@@ -27,20 +22,14 @@ remove:
   - cups-filters
   - braille-printer-app
   - libppd
-  - plasma-discover-flatpak
-  - ibus
-  - ibus-panel
   - ibus-m17n
   - ibus-libpinyin
   - ibus-hangul
   - ibus-libzhuyin
   - ibus-qt
   - ibus-anthy
   - ibus-anthy-python
-  - ibus-setup
   - ibus-typing-booster
   - open-vm-tools
   - open-vm-tools-desktop
-  - krfb
```

Comment:
flatpak needs this? :(

Reply:
kinoite build fails otherwise :unamused: